src/domain/animal_detection_mode.py

```diff
@@ -29,13 +31,16 @@ def run(self):
         logger.info("Instantiating cameras...")
         camera: Camera
         for camera in cameras:
-            if not camera.enabled or camera.type != Camera.CameraTypes.USBCamera:
-                continue
-            else:
+            if camera.enabled and camera.type == Camera.CameraTypes.USBCamera:
                 # Create thread for motion detection
                 logger.debug("Instantiating thread for camera %s", camera.id)
                 camera.stop_thread = False
                 self.camera_thread.append(camera.run())
+            elif camera.enabled and camera.type == Camera.CameraTypes.FTPCamera and FTPsServer.ftps_server:
```

Comment:
You've already checked camera.enabled on the if statement. Maybe you can reorganize the ifs like:

if camera.enabled:
    if camera.type == Camera.CameraTypes.USBCamera:
    [...]
    elif camera.type == Camera.CameraTypes.FTPCamera: